### DISCUSSION TITLE
Can't get RTSP to work

### DISCUSSION THREAD

**Discussion by alex-rsk** [bluenviron/mediamtx#2459] [Questions]
### Question

Hello, i stuck with simple task: I have some RTSP stream, I want to read it and broadcast through mediamtx.
My config file is:

```
  paths:
    cam1:
       source: rtsp://zephyr.rtsp.stream/pattern?streamKey=87a01fc49801f9771ecd0bb7666d6c04
```
The stream is active and I can watch it with VLC player.

I got:
`closed: no one is publishing to path 'cam1'`
'
Tested with several 100% working RSTP streams and still getting this errror.

What am I doing wrong?

**Comment by aler9** [comment]
Hello, the configuration seems right to me, that error is usually shown when the server can't pull the stream from the camera. If you still have problems, post the full server log, that generally contains the precise error.

[a comment by an automated account is not shown]
